specs/interop/README.md

```diff
@@ -0,0 +1,81 @@
+- **Credential Format** – what type of Verifiable Credential(s) to be used (e.g. W3C Verifiable Credentials v1.1, v2.0, Open ID Tokens, SD-JWT/SD-JWT-VC, mDoc/mDL, etc.).
+- **User Authentication** – how one *authenticates* with other entities (e.g. DIDAuth, OpenID Connect, SIOPv2, etc.).
+- ********************************************Credential Application******************************************** – how entities apply for and receive Verifiable Credentials (e.g. VC-API, OID4VCI, Credential Manifest, a DWN protocol, etc.).
+    - **Note:** There is a distinction between Credential Application protocols like OID4VCI and Credential Application languages/data models like Credential Manifest.
+- **Credential** **Presentation** – how entities request and share presentations of credentials with one another (e.g. VC-API, OID4VP, Presentation Exchange, a DWN protocol, etc.).
+    - **Note:** There is a distinction between Credential Presentation protocols like OID4VP and Credential Presentation languages/data models like Presentation Exchange.
+- **Credential Status** – how issuers, post-issuance, can represent the status of a Verifiable Credential (e.g. Status List 2021, etc.).
+- **tbDEX Messages** – how messages in tbDEX interactions look and are sent (e.g. data model, protocol, transport).
+
+## Existing Profiles
+
+There are a number of profiles in the space, here are are a few prominent ones:
+
+- (DIF) [DIF JWT VC Presentation Profile](https://identity.foundation/jwt-vc-presentation-profile/)
+- (HAIP) [OpenID4VC High Assurance Interoperability Profile with SD-JWT VC](https://vcstuff.github.io/oid4vc-haip-sd-jwt-vc/draft-oid4vc-haip-sd-jwt-vc.html)
+- (EBSI) [European Blockchain Standards Initiative Interop Profile](https://ec.europa.eu/digital-building-blocks/wikis/display/EBSIDOC/)
+- (MSFT) [Microsoft Supported standards](https://learn.microsoft.com/en-us/entra/verified-id/verifiable-credentials-standards)
+
+There have been recent conversations at IIW, in the OIDF, and in DIF about updating and continuing to standardize on these profiles. In the meantime, we need to deliver working software that facilitates interoperability. As such, let’s work on our own profile that leverages the best parts of existing profiles, since none exactly fit all the features we need to support.
+
+## A tbDEX Profile
+
+Our first tbDEX profile does not prevent there from being other profiles, nor does it restrict what types of interactions are permitted on tbDEX. Instead, the profile aims to promote interoperability between implementers that can be built and iterated upon.
+
+With tbDEX we identify three main parties. The parties may overlap (e.g. a tbDEX node can be a credential issuer).
+
+- **tbDEX nodes** — run to facilitate tbDEX messages, interacts with end-users through web applications, wallets, etc.
```

Comment:
Does the concept of a `tbDEX node` cover a PFI's implementation (ex. a [TypeScript HTTP Server](https://github.com/TBD54566975/tbdex-js/tree/main/packages/http-server)) **and** also a Wallet (AKA "Alice" -- ex. a [TypeScript HTTP Client](https://github.com/TBD54566975/tbdex-js/tree/main/packages/http-client))? Or, said differently, `tbdex node` is headless tech which can be contrasted w/ "user-centric" (AKA GUIs) `Wallets/Applications` (below).

---

(from above)
> With tbDEX we identify three main parties

The three parties I have categorized to-date are:

1. PFIs
2. Issuers
3. Wallets/Applications (I have some gripes with this word but it's okay for now)

Reply:
agree with your categorization. I think the latest one I'd like to call "user agent"

Reply:
does user agent have any inherent coupling with DWNs, or is the term used outside of that as well?

Reply:
the intention is to not couple with DWNs, perhaps generic "agent" is good and we can add a sentence or two describing what it means

Reply:
In favor of just "agent" 👍 

![image](https://github.com/TBD54566975/tbdex/assets/3602993/008d15f0-8807-4c37-be71-9aa68c1a8099)